en/mulesoft/modules/ROOT/pages/ms_composer_salesforce_marketing_cloud_reference.adoc

```diff
@@ -2,6 +2,43 @@
 
 Salesforce Marketing Cloud is a marketing platform that supports capabilities such as real-time customer engagement, email and marketing automation, social media engagement, mobile (SMS) messaging and push notifications, and customer marketing analytics.
 
+
+== Create API Integration Package and Set Permissions
+
+To use the Salesforce Marketing Cloud Connector, you must first create an API Integration package and set permissions for email and SMS communication.
```

Comment:
I think this sentence fits better in the introductory paragraph, working like a pre-requisite or "before you begin" using the Connector.

Reply:
Good suggestion - Done!